src/main/java/org/terasology/scenario/internal/events/ConvertScenarioEntityEvent.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.scenario.internal.events;
+
+import org.terasology.entitySystem.event.Event;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ConvertScenarioEntityEvent implements Event {

```

Comment:
This class needs a javadoc badly, to avoid explaining things twice you can just mention the role of the class and then reference to one class that has the complete explanation.